cabal-install/Distribution/Client/Compat/Directory.hs

```diff
@@ -0,0 +1,14 @@
+{-# LANGUAGE CPP #-}
+module Distribution.Client.Compat.Directory (setModificationTime) where
+
+#if MIN_VERSION_directory(1,2,3)
+import System.Directory (setModificationTime)
+#else
+
+import Data.Time.Clock (UTCTime)
+import System.IO (FilePath)
```

Comment:
IIRC `FIlePath` is in `Prelude`: http://hackage.haskell.org/package/base-4.10.1.0/docs/Prelude.html#t:FilePath

Reply:
Yep. Fixed.